_posts/2018-04-15-lazy-loading-images.md

```diff
@@ -0,0 +1,205 @@
+---
+title: 'Improve pagespeed: Lazy Loading Images'
+author: dimitri-steinel
+layout: post
+mainImage: ""
+mainImageAlt: ""
+--- 
+
+Don't want to read it all? [Here’s the code](#snippet)
+
+### First meaningful paint
+
+Have you heard of first meaningful paint? No? You should! The first meaningful paint is a very important parameter of you website. The longer a page needs to load the more the users lose interest and exit your website. So there is a very close connection between pagespeed and bounce rate.
```

Comment:
This sentence could be re-written as: `The longer a page needs to load the more likely as user is to lose interest and exit your website.`